spec/resty/concurrent/executor_spec.lua

```diff
@@ -0,0 +1,15 @@
+local Executor = require('resty.concurrent.executor')
+
+describe('ImmediateExecutor', function()
+    describe('.from_options', function()
+        it('has default executor', function()
+            assert(Executor.from_options{ })
+        end)
+
+        it('has passes through executor', function()
+            local executor = { 'something' }
+
+            assert.equal(executor, Executor.from_options{ executor = executor })
+        end)
+    end)
```

Comment:
I think we should also try with strings.  `Executor.from_options { executor = _one_that_exists_ })` and with `_one_that_does_not_exist_`